### PR TITLE
Dataworks aws ingest reconcilers

### PR DESCRIPTION
**Pull request by mark-r-m**
Dataworks aws ingest reconcilers

### CODE REVIEW

File: dataworks-aws-ingest-reconcilers.tf [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+  triggers = {
+    repo = github_repository.dataworks_aws_ingest_reconcilers.name
+  }
+  provisioner "local-exec" {
+    command = "./initial-commit.sh ${github_repository.dataworks_aws_ingest_reconcilers.name} '${github_repository.dataworks_aws_ingest_reconcilers.description}' ${github_repository.dataworks_aws_ingest_reconcilers.template.0.repository}"
+  }
+}
+
+locals {
+  dataworks_aws_ingest_reconcilers_pipeline_name = "aws-ingest-reconcilers"
```
Comment (dan-hill2802) [inline dataworks-aws-ingest-reconcilers.tf:60]: probably want to drop the `aws-` prefix

File: dataworks-aws-ingest-reconcilers.tf [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+}
+resource "github_repository_webhook" "dataworks-aws-ingest-reconcilers" {
```
Comment (dan-hill2802) [inline dataworks-aws-ingest-reconcilers.tf:62]: resources should be `_` instead of `-`

File: dataworks-aws-ingest-reconcilers.tf (hunk 1 above)
Comment (mark-r-m) [reply dataworks-aws-ingest-reconcilers.tf:60]: With aws it matches our pattern more (See new concourse)?

File: dataworks-aws-ingest-reconcilers.tf (hunk 2 above)
Comment (mark-r-m) [reply dataworks-aws-ingest-reconcilers.tf:62]: damn cheers

**Review by dan-hill2802 — APPROVED** [review]

**Pull request merged**